src/sentry/utils/http.py

```diff
@@ -209,29 +210,40 @@ def is_valid_origin(origin, project=None, allowed=None):
     return False
 
 
-def is_valid_ip(ip_address, project):
+def is_valid_for_processing(value, filter_type, project):
     """
-    Verify that an IP address is not being blacklisted
-    for the given project.
+    Verify that an exception is valid and is not disallowed for the project
+    based on specified filters
+
+    value is the exact value to be checked for validity in the list of project filters
+
+    filter_type is the type of filter and can be:
+    'blacklisted_ips'
+    'releases'
+    'error_messages'
     """
-    blacklist = project.get_option('sentry:blacklisted_ips')
-    if not blacklist:
+
+    disallowed_options = project.get_option(('sentry:{}').format(filter_type))
+    if not disallowed_options:
         return True
 
-    for addr in blacklist:
+    for option in disallowed_options:
         # We want to error fast if it's an exact match
-        if ip_address == addr:
+        # fnmatch allows us to use glob pattern matching
+        # so that we can match on 1.* or TypeError*
```

Comment:
fnmatch covers a few other cases as well. And if we document, would be worth noting.

```python
>>> fnmatch.fnmatch('foo', 'f?o')
True
>>> fnmatch.fnmatch('foo', '[fo]oo')
True
>>> fnmatch.fnmatch('foo', '[fx]oo')
True
>>> fnmatch.fnmatch('xoo', '[fx]oo')
True
>>> fnmatch.fnmatch('yoo', '[!x]oo')
True
```